app/helpers/version_contents_helper.rb

```diff
@@ -0,0 +1,17 @@
+module VersionContentsHelper
+  def folder_icon
+    <<~SVG.html_safe
+      <svg height="16" width="16" viewBox="0 0 16 16">
+        <path d="M 2 1 L 0 3 v 11 L 1 15 h 14 L 16 14 v -10 L 15 3 H 10 L 8 1 Z"></path>
```

Comment:
Credit to [this awesome little svg editor](https://yqnn.github.io/svg-path-editor/) that allowed me to throw together some icons without stealing anything from anyone else. We could refine these, obviously, or use better designs, but I was thoroughly impressed with how easy it was to make a couple quick icons. Thank you @yqnn!